lib/generators/sorcery/install_generator.rb

```diff
@@ -97,11 +97,7 @@ def only_submodules?
       end
 
       def migration_class_name
-        if Rails::VERSION::MAJOR >= 5
-          "ActiveRecord::Migration[#{Rails::VERSION::MAJOR}.#{Rails::VERSION::MINOR}]"
-        else
-          'ActiveRecord::Migration'
-        end
+        "ActiveRecord::Migration[#{Rails::VERSION::MAJOR}.#{Rails::VERSION::MINOR}]"
```

Comment:
I think we might be able to refactor this to just use `ActiveRecord::Migration::Current`, but non-blocking.

Reply:
@joshbuker When using ActiveRecord::Migration::Current here, the content of the migration file might change when users upgrade their Rails version, so I think it's best to leave it as is.

Reply:
Wouldn't that also be true using this method?